v2/test/components/kaniko.yaml

```diff
@@ -0,0 +1,57 @@
+name: Kaniko
+inputs:
+- {name: dockerfile, type: String}
+- name: context_uri
+  type: URI
+  default: ""
+  description: |
+    URI to download context, e.g. gs://mybucket/context.tar.gz.
+    At least one of context_uri and context_artifact should be specified.
+- name: context_artifact
+  type: Folder
+  default: ""
+  description: |
+    Input context artifact, it should be a folder.
+    At least one of context_uri and context_artifact should be specified.
+- {name: context_sub_path, type: Path, default: ""}
+- {name: destination, type: URI, description: destination should not contain tag}
+outputs:
+- {name: digest, type: URI, description: Image URI with full digest}
+metadata:
+  annotations:
+    author: Yuan Gong <gongyuan94@gmail.com>
+implementation:
+  container:
+    # The debug image contains busybox, while the normal images do not.
```

Comment:
Great find. I've hit this problem as well when creating a Kaniko component for the tutorials.